temba/channels/models.py

```diff
@@ -2013,10 +2013,8 @@ def send_vumi_message(cls, channel, msg, text):
         is_ussd = channel.channel_type in Channel.USSD_CHANNELS
         channel.config['transport_name'] = 'ussd_transport' if is_ussd else 'mtech_ng_smpp_transport'
 
-        if msg.response_to_id:
-            in_reply_to = Msg.objects.get(pk=msg.response_to_id).external_id
-        else:
-            in_reply_to = None
+        in_reply_to = Msg.objects.values_list(
+            'external_id', flat=True).get(pk=msg.response_to_id)
```

Comment:
@nullcode this is actually straight out of the Django docs and the recommended way of getting a single value from a FK:
  https://docs.djangoproject.com/en/1.10/ref/models/querysets/#values-list

Since it is a foreign key and we never delete incoming messages the lookup should never fail from a GET perspective.